client/landing/gutenboarding/utils/analytics.ts

```diff
@@ -0,0 +1,47 @@
+/**
```

Comment:
I think that this file should live outside of utils because as far as I can see, `utils/` are only pure functions and the `recordTracksEvent` is a side effect

Reply:
Hmm, that is a good point about side effects. Do you reckon it's worth moving it into a store with action/controller type setup (in another PR)? Or just create a new `analytics` dir for now?

Reply:
hmm I was thinking just to put it at the top level alongside `path.ts`, as far as I can tell that seems to be where these kind of common files have been kept in other parts of the codebase.  Or I've seen a `lib` folder being used too like we have `client/lib/analytics/`. Or yea just an `analytics` dir from the top level would work too.

Reply:
yea I think best would be like `gutenboarding/lib/analytics/analytics.ts` and then later we could move any other shared code with side effects there like `path.ts`.

Reply:
Ah cool. I'll move it into `gutenboarding/lib/analytics/index.ts` for now. You're right: we can worry about where to dump it later 🤣